pom.xml

```diff
@@ -617,6 +617,21 @@
                 <artifactId>jsoup</artifactId>
                 <version>1.15.3</version>
             </dependency>
+
+
+            <!-- Testcontainers -->
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>junit-jupiter</artifactId>
+                <version>1.17.3</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>mongodb</artifactId>
+                <version>1.17.3</version>
```

Comment:
The `version` properties should use the `testcontainers.version` property.

Reply:
Done.